docs/website/content/v0.6/en/guides/upgrading/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Upgrading
+---
+
+In Kubernetes 1.19, the official `hyperkube` image was removed.
+This means that in order to upgrade Kubernetes, Talos users will have to change the `command` field of each control plane component.
+The `kubelet` image will also have to be updated, if you wish to specify the `kubelet` image explicitly.
+The default used by Talos is sufficient in most cases.
+
+### API Server
```

Comment:
should we provide some pointers that one has to download `kubeconfig` and use `kubectl` for that?